plasma_framework/contracts/src/exits/payment/PaymentExitDataModel.sol

```diff
@@ -39,8 +41,10 @@ library PaymentExitDataModel {
          */
         uint256 exitMap;
         uint256 position;
-        PaymentOutputModel.Output[MAX_INPUT_NUM] inputs;
-        PaymentOutputModel.Output[MAX_OUTPUT_NUM] outputs;
+        // TODO: need to save the output guard during IFE
+        bytes32[MAX_INPUT_NUM] outputGuardForInputs;
```

Comment:
@pgebal I end up still need the `outputGuard` for inputs, ah.........(just realized that getting output guard from unknown tx type is......fun/complex, so I think it might be better to do that only in startIFE)

Now I re-think, probably it worth to keep inconsistence and let the input save the exitTarget so we don't need extra storage? : (

Reply:
@boolafish ok, I'll code that tomorrow. I'll change WithdrawData to store exit target instead of output guard. For in-flight transaction outputs I'll store only outputId, leaving exitTarget with default value of zeroes.